tensorflow/lite/python/interpreter_wrapper/interpreter_wrapper.cc

```diff
@@ -259,6 +259,7 @@ InterpreterWrapper::~InterpreterWrapper() {}
 PyObject* InterpreterWrapper::AllocateTensors(int subgraph_index) {
   TFLITE_PY_ENSURE_VALID_INTERPRETER();
   TFLITE_PY_SUBGRAPH_BOUNDS_CHECK(subgraph_index);
+  TFLITE_PY_CHECK(interpreter_->ApplyLazyDelegateProviders());
```

Comment:
Thanks for spotting the issue in Python! However, we think it's better not to call this private function here. We are preparing an alternative fix now, and we will get it out asap.